src/setupProxy.js

```diff
@@ -1,13 +1,13 @@
-const proxy = require('http-proxy-middleware');
+const {createProxyMiddleware} = require('http-proxy-middleware');
```

Comment:
I know this change may cause us trouble if we do not update the http-proxy-middleware dependency. It was a breaking change.

Reply:
We need to let the whole team know that, once we merge these changes in, they will definitely have to delete node_modules and reinstall

Reply:
Note by the title that this is a merge of stuff from develop into our branch. So these were changes already reviewed and approved before they got merged into develop, and not written by me

Reply:
Ah, I now understand what you mean in the second comment, that is definitely a good point, that we would expect the existing 'npm install' of our local clones of the repo to break once this branch gets merged into apartapp-develop.

Reply:
right, thanks for trying to understand!